packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -418,7 +418,8 @@ class ApplicationCommand extends Base {
         command.descriptionLocalizations ?? command.description_localizations ?? {},
         this.descriptionLocalizations ?? {},
       ) ||
-      !isEqual(command.integrationTypes ?? command.integration_types ?? [], this.integrationTypes ?? []) ||
+      // [0] is the default value sent by Discord
+      !isEqual(command.integrationTypes ?? command.integration_types ?? [0], this.integrationTypes ?? [0]) ||
```

Comment:
Let's use the enum for the number, please, so it's clearer what it represents

Reply:
Just to confirm, is it `ApplicationIntegrationType.GuildInstall`?

Reply:
Yep. See [here](https://discord.com/developers/docs/resources/application#application-object-application-integration-types).